### PR TITLE
Fingerprint: add language attributes in mark-up

### PR DESCRIPTION
**Pull request by wiese**
Bug: T213926

### CODE REVIEW

**Comment by wiese** [comment]
~~Goes into #182~~

**Comment by wiese** [comment]
Rebased

File: tests/unit/components/Fingerprint.spec.ts [hunk 1]
```diff
@@ -24,6 +26,9 @@ const entityAliasesDe = [ 'Antwort auf alles', 'Ihr kennt ja nicht einmal die Fr
 
 const languageCodeWithoutDataInEntity = 'en';
 
+const languageAr = { code: languageCodeAr, directionality: 'rtl' } as Language;
```
Comment (jakobw) [inline tests/unit/components/Fingerprint.spec.ts]:
```
|￣￣￣￣￣￣￣￣￣￣￣| 
|    UNNECESSARY   | 
|     TYPE         | 
|   ASSERTION      | 
| ＿＿＿＿＿＿＿＿＿＿| 
(\__/) || 
(•ㅅ•) || 
/ 　 づ
```

```suggestion
const languageAr = { code: languageCodeAr, directionality: 'rtl' };
```

File: tests/unit/components/Fingerprint.spec.ts [hunk 2]
```diff
@@ -24,6 +26,9 @@ const entityAliasesDe = [ 'Antwort auf alles', 'Ihr kennt ja nicht einmal die Fr
...
+const languageDe = { code: languageCodeDe, directionality: 'ltr' } as Language;
```
Comment (jakobw) [inline tests/unit/components/Fingerprint.spec.ts]:
```suggestion
const languageDe = { code: languageCodeDe, directionality: 'ltr' };
```

File: tests/unit/components/Fingerprint.spec.ts [hunk 3]
```diff
@@ -8,12 +8,14 @@ import {
 } from '@/store/namespaces';
 import { ENTITY_INIT } from '@/store/entity/mutationTypes';
 import { LANGUAGE_INIT } from '@/store/user/mutationTypes';
-import { LANGUAGE_TRANSLATION_UPDATE } from '@/store/language/mutationTypes';
+import { LANGUAGE_TRANSLATION_UPDATE, LANGUAGE_UPDATE } from '@/store/language/mutationTypes';
 import FingerprintableEntity from '@/datamodel/FingerprintableEntity';
 import { mutation } from '@/store/util';
+import Language from '@/datamodel/Language';
```
Comment (jakobw) [inline tests/unit/components/Fingerprint.spec.ts:13]:
```suggestion
```

**Review by jakobw — COMMENTED** [review]
Nitpick, rest looks good!

File: tests/unit/components/Fingerprint.spec.ts (hunk 1 above)
Comment (wiese) [reply tests/unit/components/Fingerprint.spec.ts]: bildschirmfoto von 2019-01-17 16-46-16
:cry:

**Pull request merged**